internal/resources/package/data_source_package.go

```diff
@@ -40,7 +40,7 @@ var packageSchema = map[string]*schema.Schema{
 	},
 	namespaceKey: {
 		Type:        schema.TypeString,
-		Description: "Namespae of package.",
+		Description: "Namespace of package.",
```

Comment:
this should have the corresponding doc changes too right?

Reply:
Ideally the schema and doc changes should go in one PR itself. Ensure to do it from next time. Approving this time considering the release timeline

Reply:
There seems to be issue with `tfplugindocs` used for the doc generation, as the description from the schema is not getting picked, instead it is removing it all.


That's the reason the doc for this is not reflecting